pages.es/windows/cd.md

```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Muestra información sobre el directorio actual y le permite moverse a otro.
+> Más informaciones: <https://learn.microsoft.com/windows-server/administration/windows-commands/cd>.
+
+- Muestra la ruta (path) del directorio actual:
+
+`cd`
+
```

Comment:
Seems like this page is missing an example from the [original page](https://github.com/tldr-pages/tldr/blob/main/pages/windows/cd.md).

```suggestion

- Ir a la raíz de la unidad actual:

`cd \`

```
Feel free to reword the suggestion I made using Google translate.